components/usage/pkg/apiv1/billing.go

```diff
@@ -67,6 +67,37 @@ func (s *BillingService) UpdateInvoices(ctx context.Context, in *v1.UpdateInvoic
 	return &v1.UpdateInvoicesResponse{}, nil
 }
 
+func (s *BillingService) ReconcileInvoices(ctx context.Context, in *v1.ReconcileInvoicesRequest) (*v1.ReconcileInvoicesResponse, error) {
+	balances, err := db.ListBalance(ctx, s.conn)
+	if err != nil {
+		log.WithError(err).Errorf("Failed to reconcile invoices.")
+		return nil, status.Errorf(codes.Internal, "Failed to reconcile invoices.")
+	}
+
+	creditSummaryForTeams := map[string]stripe.CreditSummary{}
+	for _, balance := range balances {
+		entity, id := balance.AttributionID.Values()
+
+		// TODO: Support updating of user attribution IDs
+		if entity != db.AttributionEntity_Team {
```

Comment:
why do we need this exclusion here?

Reply:
Legacy of the focus on team based UBP first.